server/subscription.go

```diff
@@ -68,6 +52,32 @@ func (s *Server) registerSubscriptionRoutes(g *echo.Group) {
 	})
 }
 
+// loadLicense will load current subscription by license.
+// Return subscription with free plan if no license found.
+func (server *Server) loadSubscription() *enterprise.Subscription {
+	subscription := &enterpriseAPI.Subscription{
+		Plan: api.TEAM,
+		// -1 means not expire, just for free plan
+		ExpiresTs:     -1,
+		InstanceCount: 5,
+	}
+	license, _ := server.loadLicense()
+	if license != nil {
+		subscription = &enterpriseAPI.Subscription{
+			Plan:          license.Plan,
+			ExpiresTs:     license.ExpiresTs,
+			InstanceCount: license.InstanceCount,
+		}
+	} else {
+		// increase instance quota for dev environment.
+		if server.mode == "dev" {
```

Comment:
I am thinking for dev mode, we just return a TEAM plan with ExpiresTs: -1 and instanceCount 9999 ?

And in frontend for dev, we design a way to let developer switch between free and team plan to test the different logic.

wdyt?

Reply:
Sure, I'd like to implement this in next PR